unrpyc.py

```diff
@@ -346,10 +346,24 @@ def main():
 
     args = ap.parse_args()
 
+    # Catch impossible arg combinations with clear info, so they do not produce unclear
+    # errors or fail silent
+    if (args.no_pyexpr or args.comparable) and not args.dump:
+        raise ap.error(
+            "Arguments 'comparable' and 'no_pyexpr' are not usable without 'dump'.")
+        return
+
+    if ((args.try_harder or args.dump)
+            and (args.write_translation_file or args.translation_file or args.language)):
+        raise ap.error(
+            "Arguments 'try_harder' and/or 'dump' are not usable with the translation "
+            "feature.")
+        return
+
+    # Fail early to avoid wasting time going through the files
     if (args.write_translation_file
-        and not args.clobber
+            and not args.clobber
             and args.write_translation_file.exists()):
-        # Fail early to avoid wasting time going through the files
         print("Output translation file already exists. Pass --clobber to overwrite.")
```

Comment:
This would gain from being turned into a raise too - but it's a bit outside the boundaries, maybe ask for Censored's confirmation

Reply:
I would be ok with the idea. Lets see what CU thinks.

Reply:
I'd just make it a raise as well.

Reply:
Ok, will add this.